multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/ndarray/operations/IteratingNDArray.kt

```diff
@@ -715,6 +716,47 @@ public inline fun <T, D : Dimension, reified R : Any> MultiArray<T, D>.map(trans
     return NDArray(data, shape = shape, dim = dim)
 }
 
+
+/**
+ * Returns the element-wise minimum of array elements for [this] and [other].
+ */
+public fun <T, D : Dimension> MultiArray<T, D>.minimum(other: MultiArray<T, D>): NDArray<T, D> {
+    requireArraySizes(this.size, other.size)
+    val ret = if (this.consistent) (this as NDArray).copy() else (this as NDArray).deepCopy()
+    when (dtype) {
+            DataType.DoubleDataType -> (ret as NDArray<Double, D>).commonAssignOp(other.iterator() as Iterator<Double>) { a, b -> min (a, b) }
```

Comment:
What formatter are you using? There is usually no space after function calls: `min(a, b)`

Reply:
probably, I disabled or forgot formatter, but there is only kotlin-official-codestyle is used by myself for two last years